search_service/__init__.py

```diff
@@ -81,17 +82,20 @@ def create_app(*, config_module_class: str) -> Flask:
     api_bp.add_url_rule('/healthcheck', 'healthcheck', healthcheck)
     api = Api(api_bp)
     # Table Search API
-    # TODO: Rename endpoint to be more generic and accept a resource type so that logic can be re-used
+
     api.add_resource(SearchTableFilterAPI, '/search_table')
+    # TODO: Rename endpoint to be more generic and accept a resource type so that logic can be re-used
     api.add_resource(SearchTableAPI, '/search')
 
     # User Search API
     api.add_resource(SearchUserAPI, '/search_user')
 
     # Dashboard Search API
     api.add_resource(SearchDashboardAPI, '/search_dashboard')
+    api.add_resource(SearchDashboardFilterAPI, '/search_dashboard_filter')
 
     # DocumentAPI
+    # todo: needs to update to handle dashboard/user or other entities use cases.
```

Comment:
Question: I'm not sure what this particular API is for, but wondering how come I don't see a reference to dashboards here either.

Reply:
so these APIs are contributed by the team from Square which they use API to clean up ES which we don't follow similar pattern. But I looked at the code which is  tight with table entity. That's the reason I add the note to update the endpoint if we want to leverage(e.g support realtime tagging which will be called by FE etc )